packages/forma-36-react-components/tools/rollup.umd.config.js

```diff
@@ -0,0 +1,56 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import localResolve from 'rollup-plugin-local-resolve';
+import replace from 'rollup-plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+import babel from 'rollup-plugin-babel';
+import path from 'path';
+
+const extensions = ['.js', '.jsx', '.ts', '.tsx', '.css'];
+
+function getConfig(environment) {
```

Comment:
@denkristoffer is this configuration new or it's a rollback to what we had in the past?

Reply:
It should be a direct copy of the old file.